### DISCUSSION TITLE
Plugin compatibility

### DISCUSSION THREAD

**Discussion by hazzah12** [jamland/samplescope#189] [Ideas]
Hi, I really like the app and what you're doing! What would be really cool would be to compatibility with DAWs (music software), perhaps in the form of a plugin (VST or AAX for example) so you could drag and drop sounds straight into other software instead of having to download each individual sound. I know very little about anything to do with coding, but that is just a little idea. Either way, the app is great, keep up the great work!

**Comment by jamland** [comment]
hey @hazzah12 ! 

You have a great idea here.  I really like the idea of more seamless integration with DAW.

Wrapping it into VST/AAX itself doesn't seem to help here with drag'n'drop. 
On other hand this will create another layer of complexity because it uses different technology stack, specifically C++. I use Electron.JS framework which uses JavaScript.

But definitely I need to research  drag'n'drop functionality with current implementation.  I think it might be possible.

Thank you for the feedback! Really appreciate it
